nipype/pipeline/engine/nodes.py

```diff
@@ -1125,6 +1125,8 @@ def _make_nodes(self, cwd=None):
                     fieldvals = filename_to_list(getattr(self.inputs, field))
                 logger.debug('setting input %d %s %s', i, field, fieldvals[i])
                 setattr(node.inputs, field, fieldvals[i])
+            node._interface.num_threads = self._interface.num_threads
```

Comment:
I have no idea. But you can run `_make_nodes` and check for yourself.

I could instead add a `BaseInterface.__deepcopy__` function.